ibl_to_nwb/AlyxToNWB/alyx_to_nwb_converter.py

```diff
@@ -388,15 +396,27 @@ def create_acquisition(self):
                 else:
                     if i['name'] in ['raw.lf', 'raw.ap']:
                         for j, probes in enumerate(range(self.no_probes)):
+                            if i['data'][j].shape[1] > self.one_data.data_attrs_dump['electrode_table_length'][j]:
+                                if 'channels.rawInd' in self.one_data.loaded_datasets:
+                                    channel_idx = self.one_data.loaded_datasets['channels.rawInd'][j].data
+                                else:
+                                    warnings.warn('could not find channels.rawInd')
+                                    break
+                            else:
+                                channel_idx = np.arange(i['data'][j].shape[1]))
                             self.nwbfile.add_acquisition(
-                                TimeSeries(
+                                ElectricalSeries(
                                     name=i['name'] + '_' + self.nwb_metadata['Probes'][j]['name'],
-                                    starting_time=i['timestamps'][j][0, 1],
+                                    starting_time=np.abs(np.round(i['timestamps'][j][0, 1], 2)),
```

Comment:
why are you rounding?

Reply:
https://github.com/catalystneuro/IBL-to-nwb/pull/15#discussion_r520618282

Reply:
Please comment that